openreview/workflows/process/committee_invited_edit_reminder_process.py

```diff
@@ -1,12 +1,14 @@
 def process(client, edit, invitation):
 
     domain = client.get_group(invitation.domain)
-    committee_key = invitation.id.split('/')[-4].lower()
-    committee_invited_id= domain.content[f'{committee_key}_invited_id']['value']
-    committee_id = domain.content[f'{committee_key}_id']['value']
-    committee_declined_id = domain.content[f'{committee_key}_declined_id']['value']
-    committee_invited_response_id = domain.content[f'{committee_key}_recruitment_id']['value']
-    committee_invited_message_id = domain.content[f'{committee_key}_invited_message_id']['value']
+    committee_id = invitation.id.split('/Invited/')[0]
```

Comment:
@melisabok these were the fixes for using the `committee_role` to get invitations from the domain content. 

I also parametrized `reviewers_name` in all the templates we are using. There's a couple templates that still have `Reviewers` hard-coded, but I am not sure if we are going to eventually get rid of those templates.